### PR TITLE
Add the Successors to World Forge

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Adds the Successors to World Forge.
The PR to the main game is [here](https://github.com/endless-sky/endless-sky/pull/10296).
Not to be merged until that is.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
Ready for review - everything I know of that needs to be implemented is in this PR. 
Note: the reputation change missions for the Houses are only offered on the Successor planet, to avoid spamming the job board on every Forge planet.

~~(checks fail because the Successors PR has not been merged yet, I can't change that)~~

**Comment by TheGiraffe3** [comment]
@Daeridanii1 does this look like it's merge-ready to you?

**Comment by TheGiraffe3** [comment]
@Daeridanii1 do you see anything wrong with this PR?

File: data/jobs.txt [hunk 1]
```diff
@@ -180,6 +180,18 @@ mission "Add Gegno Licenses"
...
+	job
+	source
+		attributes forge
+	description "Gives you the Successor license."
+	to offer
+		not "license: Successor"
+	on accept
+		set "license: Successor"
+		fail
+
```
Comment (Daeridanii1) [inline data/jobs.txt]:
```suggestion
mission "Add Successor Licenses"
	repeat
	job
	source
		attributes forge
	description "Gives you the Successor and High Houses licenses."
	to offer
		or
			not "license: Successor"
			not "license: High Houses"
	on accept
		set "license: Successor"
		set "license: High Houses"
		fail

```

File: data/jobs.txt [hunk 2]
```diff
@@ -210,6 +222,7 @@ mission "Remove Licenses"
 		clear "license: Scin Hoplologist"
 		clear "license: Scin Architect"
 		clear "license: Scin Adjutant"
+		clear "license: Successor"
 		fail
```
Comment (Daeridanii1) [inline data/jobs.txt:230]:
```suggestion
		clear "license: High Houses"
		fail
```

**Review by Daeridanii1 — COMMENTED** [review]
I'm not very familiar with World Forge's data style, but from a player's perspective, I think it makes sense to have the HH license available through the job board. This seems to mirror what I see in `jobs.txt` already.

**Comment by TheGiraffe3** [comment]
@warp-core is this merge-ready now that 0.10.9 has come out?

File: data/shipyards and outfitters.txt [hunk 3]
```diff
@@ -896,9 +943,25 @@ shipyard sheragi
...
+	"Kvelqde"
+	"Kvelqde-Khora"
+	"Myaede"
+	"Ojde"
+	"Seiitej"
+	"Stolsaqra"
+	"Veusade"
+	"Nnesaarau"
+	"Vujlet"
+	"Nnaatej"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]: Alphabetize
```suggestion
	"Aaulqra"
	"Ejniisde"
	"Kaskhorade"
	"Kvelqde"
	"Kvelqde-Khora"
	"Myaede"
	"Nnaatej"
	"Nnesaarau"
	"Ojde"
	"Seiitej"
	"Stolsaqra"
	"Veusade"
	"Vujlet"
```

File: data/shipyards and outfitters.txt [hunk 4]
```diff
@@ -1163,6 +1238,9 @@ shipyard allvariants
 	"'olofez (Digger)"
 	"'olofez (Expeller)"
 	"'olofez (World-Ship)"
+	"Aaulqra"
+	"Aaulqra (Brawler)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1242]:
```suggestion
	"Aaulqra (Brawler)"
	"Aaulqra (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 5]
```diff
@@ -1553,6 +1635,11 @@ shipyard allvariants
 	"Korsmanath A'awoj (Special)"
 	"Korsmanath A'awoj (Strike)"
 	"Kunai"
+	"Kvelqde"
+	"Kvelqde (Mining)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1640]:
```suggestion
	"Kvelqde (Mining)"
	"Kvelqde (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 6]
```diff
@@ -1660,13 +1747,22 @@ shipyard allvariants
 	"Mule (Hai Trafficker)"
 	"Mule (Hai Weapons)"
 	"Mule (Heavy)"
+	"Myaede"
+	"Myaede (Mining)"
+	"Myaede (Mining, Defensive)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1754]:
```suggestion
	"Myaede (Mining, Defensive)"
	"Myaede (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 7]
```diff
@@ -1660,13 +1747,22 @@ shipyard allvariants
...
 	"Nest (Miner)"
 	"Nest (Sidewinder)"
 	"Nighthawk"
 	"Nighthawk (Raider)"
 	"Nighthawk (Slaver)"
+	"Nnaatej"
+	"Nnaatej (Defensive)"
+	"Nnesaarau"
+	"Ojde"
+	"Ojde (Mining)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]:
```suggestion
	"Ojde (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 8]
```diff
@@ -1660,13 +1747,22 @@ shipyard allvariants
...
 	"Nanobot"
 	"Nest"
 	"Nest (Miner)"
 	"Nest (Sidewinder)"
 	"Nighthawk"
 	"Nighthawk (Raider)"
 	"Nighthawk (Slaver)"
+	"Nnaatej"
+	"Nnaatej (Defensive)"
+	"Nnesaarau"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1766]:
```suggestion
	"Nnesaarau"
	"Nnesaarau (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 9]
```diff
@@ -1822,6 +1919,10 @@ shipyard allvariants
 	"Starling (Keystone)"
 	"Starling (Sniper)"
 	"Starling (Thrasher)"
+	"Stolsaqra"
+	"Stolsaqra (Military)"
+	"Stolsaqra (Mission Escort)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1928]:
```suggestion
	"Stolsaqra (Mission Escort)"
	"Stolsaqra (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 10]
```diff
@@ -1864,10 +1965,15 @@ shipyard allvariants
 	"Vareti'het (EMP, Disable-able)"
 	"Vareti'het (Strong)"
 	"Vareti'het (Strong, Disable-able)"
+	"Veusade"
+	"Veusade (Surveillance)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1975]:
```suggestion
	"Veusade (Naive)"
	"Veusade (Surveillance)"
```

File: data/shipyards and outfitters.txt [hunk 11]
```diff
@@ -1864,10 +1965,15 @@ shipyard allvariants
...
 	"Violin Spider"
 	"Violin Spider (Pulse)"
 	"Void Sprite"
 	"Void Sprite (Infant)"
+	"Vujlet"
+	"Vujlet (Aggressive)"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:1981]:
```suggestion
	"Vujlet (Aggressive)"
	"Vujlet (Naive)"
```

File: data/shipyards and outfitters.txt [hunk 12]
```diff
@@ -1888,6 +1994,12 @@ shipyard allvariants
 	"vyu-Ir"
 
 outfitter alloutfits
+	`1-"Uurau" Betavoltaic Cell`
+	`3-"Kaska" Betavoltaic Cell`
+	`6-"Ruuv" Betavoltaic Cell`
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]: I need these further down.
```suggestion
```

File: data/shipyards and outfitters.txt [hunk 13]
```diff
@@ -1902,6 +2014,25 @@ outfitter alloutfits
...
+	`OC-"Mavra" Betavoltaic Cell`
+	`"Ojet" Bimodal Coilgun`
+	`"Ryuit" Fusion Core`
+	`"Sjeja" Kinetic Lance`
+	`"Tuur" Scanning Suite`
+	`"Uoret" Fusion Core`
+	`"Veldt" Combat Scanner`
+	`"Veusa" Inductive Reverser`
+	`"Veusa" Inductive Steering`
+	`"Veusa" Inductive Thruster`
```
Comment (warp-core) [inline data/shipyards and outfitters.txt:2038]:
```suggestion
	`"Veusa" Inductive Thruster`
	`1-"Uurau" Betavoltaic Cell`
	`3-"Kaska" Betavoltaic Cell`
	`6-"Ruuv" Betavoltaic Cell`
```

File: data/shipyards and outfitters.txt [hunk 14]
```diff
@@ -2116,6 +2250,9 @@ outfitter alloutfits
 	"Heliarch License"
 	"Heliarch Repulsor"
 	"Hexagonal Crystal"
+	"High-Velocity Spike"
+	"High-Velocity Spike Rack"
+	"High Houses License"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]:
```suggestion
	"High Houses License"
	"High-Velocity Spike"
	"High-Velocity Spike Rack"
```

File: data/shipyards and outfitters.txt [hunk 15]
```diff
@@ -2271,6 +2413,8 @@ outfitter alloutfits
 	"Outfits Expansion"
 	"Outskirts Gauger"
 	"Overcharged Shield Module"
+	"Overcharged Laser Turret"
+	"Overcharged Pulse Laser"
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]:
```suggestion
	"Overcharged Laser Turret"
	"Overcharged Pulse Laser"
	"Overcharged Shield Module"
```

File: data/shipyards and outfitters.txt [hunk 16]
```diff
@@ -1902,6 +2014,25 @@ outfitter alloutfits
...
+	`"Byiaa" Shield Cortex`
+	`"Chyyra" Inductive Reverser`
+	`"Chyyra" Inductive Steering`
+	`"Chyyra" Inductive Thruster`
+	`"Ej" Shield Cortex`
+	`"Jaase" Shield Cortex`
+	`"Jorv" Shield Cortex`
+	`"Niis" Inductive Engines`
+	`"Nnesa" Fusion Core`
+	`OC-"Mavra" Betavoltaic Cell`
```
Comment (warp-core) [inline data/shipyards and outfitters.txt]: This should be immediately after "Nuclear Missile".

**Comment by TheGiraffe3** [comment]
I had somehow assumed that nobody would want to buy the Naive variants of any ship.

File: data/map.txt [hunk 17]
```diff
@@ -729,7 +730,62 @@ system Remnant
 		distance 1930
 		period 1598.78
 
-
+system Successor
+	pos -9820.108 -10035.952
+	habitable 955.249
+	belt 1535 9
+	belt 2444
+	haze star/void_scar
```
Comment (warp-core) [inline data/map.txt]: I don't know what this is supposed to be.
The file is named "void-scar" and it's used as a star sprite, not haze.

File: data/map.txt [hunk 18]
```diff
@@ -729,7 +730,62 @@ system Remnant
...
+		distance 7524
+		period 8446.464
+		object
+			sprite planet/europa
+			distance 408
+			period 389.854
+		object
+			sprite planet/rock14-b
+			distance 550
+			period 337.568
```
Comment (warp-core) [inline data/map.txt]: We usually copy the relevant capital/home system. So, here it'd be something like:
```suggestion
system Successor
	pos -9820.108 -10035.952
	arrival 3450
	habitable 3450
	belt 1179 3
	haze _menu/haze-67
	link Remnant
	asteroids "large metal" 10 3.062
	object
		sprite star/m-giant
		period 10
	object
		sprite planet/gas6-b
		distance 1311
		period 323.262
	object
		distance 2153
		period 680
		object
			sprite planet/ice8
			distance 120
			period 140
		object Successor
			sprite planet/dust0-b
			distance 120
			period 140
			offset 180
	object
		sprite planet/browndwarf-t
		distance 2804
		period 1011.154
	object
		sprite planet/neptune
		distance 4811
		period 2272.498
		object
			sprite planet/miranda
			distance 238
			period 173.691
	object
		sprite planet/jupiter
		distance 9120
		period 5931.202
		object
			sprite planet/desert4-b
			distance 263
			period 201.765
	object
		sprite planet/browndwarf-y
		distance 11064
		period 7925.354
		object
			sprite planet/rock7-b
			distance 293
			period 237.254
		object
			sprite planet/oberon-b
			distance 398
			period 489.011
```

File: data/map.txt [hunk 19]
```diff
@@ -1267,6 +1323,19 @@ planet Remnant
 	outfitter basic
 	security 0
 
+planet Successor
+	attributes forge forgesuccessor
+	landscape land/water22
```
Comment (warp-core) [inline data/map.txt]: We don't use normal landscape images.
```suggestion
```

File: data/map.txt (hunk 17 above)
Comment (TheGiraffe3) [inline data/map.txt:738]:
```suggestion
```

**Comment by warp-core** [comment]
Thanks!

**Pull request merged**